DamienDeSousa/symfony-cms
# Display a site on the admin section

* display site in french and english
 * display a specific page if no site
 * add translation for messages in fr and en
 * add test on no site controller page
 * remove captcha test to avoid captcha issues for other tests
 * remove unused functional_test
 * enable user account for tests
 * use plaintext for test to avoid encrypt
 * create a twig function to get site title in template
 * remove unused and duplicated var in test env
 * desable mock session to enable session in test env
 * add test to verify the title of each admin page
 * add test to verify the site informations displayed in admin page
